packages/contracts/contracts/deploy/PayableDAppSystem.sol

```diff
@@ -43,4 +59,16 @@ contract PayableDAppSystem is IPayableDAppSystem {
 
         return payableFactory;
     }
+
+    function _createPaymentSplitter(
+        address _payee
```

Comment:
This parameter `_payee` can be confused with `payee`.
Should we rename it? `_otherPayee`?